apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -42,6 +42,146 @@ int main(int argc, char **argv)
 	fprintf(stdout, "%15s: Hello, World!! on stdout\n", argv[0]);
 	fprintf(stderr, "%15s: Hello, World!! on stderr\n", argv[0]);
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	volatile uint32_t *address;
```

Comment:
How about creating a separate entry under elf instead of adding MPU test in same app?

Reply:
I think its better not to create a separate app for testing the MPU.. It will add overhead of another RAM partition and things will get complicated.